include/smb2/libsmb2.h

```diff
@@ -76,13 +76,22 @@ struct smb2dirent {
         struct smb2_stat_64 st;
 };
 
-#ifdef _MSC_VER
+#if defined(_WINDOWS)
 #include <winsock2.h>
+#elif defined(_XBOX)
+#include <xtl.h> 
+#include <winsockx.h>
+#endif
+
+#if defined(_WINDOWS) || defined(_XBOX)
+typedef SOCKET t_socket;
+#elif defined(_XBOX)
```

Comment:
This logic seems odd.  
Should line 86 be "#if defined(_WINDOWS) && !defined(_XBOX)"
or something?

Reply:
Yep there's some mistakes

Reply:
Solved.